### ISSUE TITLE
DOCS: monitoring.properties.s3

### ISSUE THREAD

**Issue by JohnPreston** [conduktor/conduktor-platform#119]
Hello Conduktor team,

Congrats on the new docs site. Looking nice. Not sure how one publicly contributes to these though.

In these [docs](https://docs.conduktor.io/platform/configuration/env-variables#monitoring-properties) I found the monitoring properties.
I haven't found any documentation that explains what this is for. Also, there are two properties,
* `monitoring.storage.s3.secretAccessKey`
* `monitoring.storage.s3.accessKeyId`

Which technically can be non-mandatory with the AWS SDK using credentials from the "default chain".
If that is indeed the case, I'd recommend adding a mention to that.

Also optionally it could be useful to specify a roleArn to use to perform S3 API Calls.

Cheers!

**Comment by JohnPreston** [comment]
> Hi @JohnPreston the monitoring properties are still a bit experimental, we will add a more complete documentation on that in the future.
> 
> Those properties are used to enable external storage for platform kafka monitoring feature that rely on embedded [cortex](https://cortexmetrics.io/) as backend. The S3 configuration is pretty much derived from [cortex s3 storage configuration](https://cortexmetrics.io/docs/configuration/configuration-file/#blocks_storage_config), and we did not test support of using credentials from "default chain" and roleArn yet.

Right. Just pointing this out, you cannot expect users on AWS to start creating IAM users & keys when the application is running with an IAM role/instance profile already.

**Comment by qboileau** [comment]
You are right, we will investigate further to see if there is a potential issue.
As always, thank you for your feedback :)
